api.raml

```diff
@@ -443,17 +443,6 @@ types:
               type: UserResponse
           204:
             description: No user record was found with that `employee_id`.
-          409:
```

Comment:
I'm surprised to see this removed...am I missing something?

Reply:
My mistake. At one point I had considered changing it to 422 like all other model validations. I'll correct this.